README.md

```diff
@@ -35,16 +36,17 @@ Do you love SIMBA Chain? Do you love Hardhat? Then you're in luck! The Hardhat p
 ## Installation Overview
 The following are the general steps to get going with the SIMBA Chain Hardhat plugin. The rest of the documentation provides details on these and other steps.
 
-1. create a directory for your Hardhat project.
+1. Create a directory for your Hardhat project.
 2. cd into that directory and start an npm project.
-3. install hardhat in that project / directory. This directory, where your package.json will live, is where you will run your Hardhat commands from.
-4. install the SIMBA Chain Hardhat plugin
-5. add a 'require' or 'import' statement to your hardhat.config.js or hardhat.config.ts file, depending on which file your project has.
-6. create a simba.json file in the top level of your project, and populate that file with 'baseURL' and 'web3Suite' fields.
-7. run `npx hardhat simba help` to make sure the plugin is installed
+3. Install hardhat in that project / directory. This directory (the root of your project, where your package.json will live), is where you will run your Hardhat commands from.
+4. Install the SIMBA Chain Hardhat plugin
+5. Add a 'require' or 'import' statement to your hardhat.config.js or hardhat.config.ts file, depending on which file your project has.
+6. (Optional) create a simba.json file in the top level of your project, and populate that file with a 'baseURL' field.
+7. (Necessary if using client credentials) Create an env file (.simbachain.env, simbachain.env, or .env) in your project root, and specify client credentials as well as your SIMBA Chain base API URL.
+8. Run `npx hardhat simba help` to make sure the plugin is installed
 
 ## Prerequisites
-You should have a SIMBA Enterprise Platform Instance to communicate with. Additionally you must have a least one contract application created in the instance. To create an application, open your browser, navigate to your instance and log in using your SIMBA user account. Click on your organization -> Applications and then click on the "Add" button. Follow the on screen instructions to create your application.
+You should have a SIMBA Blocks Instance to communicate with. Additionally you must have a least one contract application created in the instance (though the plugins will direct you to create one if you do not yet have one). To create an application, open your browser, navigate to your instance and log in using your SIMBA user account. Click on your organization -> Applications and then click on the "Add" button. Follow the on screen instructions to create your application.
```

Comment:
though the plugin's' ? Does it take a 's' there?

Reply:
I was referring to the hardhat and truffle plugins. I'll change to plugin